packages/core/content-type-builder/admin/src/components/FormModal/attributes/validation/common.ts

```diff
@@ -118,7 +118,11 @@ const createTextShape = (usedAttributeNames: Array<string>, reservedNames: Array
         name: 'isValidRegExpPattern',
         message: getTrad('error.validation.regex'),
         test(value) {
-          return new RegExp(value || '') !== null;
+          try {
+            return new RegExp(value || '') !== null;
```

Comment:
When can this fail?

Reply:
Also shouldn't this use ?? to avoid replacing falsy values?

Reply:
if you pass an invalid regexp like '*' it throws I noticed that while testing so just wrapping in a try catch makes the error much better. 

value is a string so the only falsy value is empty string which is the default anyway